src/domain/common/whiteboard-rt/whiteboard.rt.resolver.fields.ts

```diff
@@ -0,0 +1,80 @@
+import { Parent, ResolveField, Resolver } from '@nestjs/graphql';
+import { Inject, LoggerService } from '@nestjs/common';
+import { WINSTON_MODULE_NEST_PROVIDER } from 'nest-winston';
+import { Profiling } from '@src/common/decorators';
+import { UseGuards } from '@nestjs/common/decorators/core/use-guards.decorator';
+import { GraphqlGuard } from '@src/core/authorization/graphql.guard';
+import { IUser } from '@domain/community/user/user.interface';
+import { EntityNotInitializedException } from '@common/exceptions/entity.not.initialized.exception';
+import { LogContext } from '@common/enums/logging.context';
+import { EntityNotFoundException } from '@common/exceptions';
+import { Loader } from '@core/dataloader/decorators';
+import { IProfile } from '../profile/profile.interface';
+import { IWhiteboardRt } from './whiteboard.rt.interface';
+import {
+  ProfileLoaderCreator,
+  UserLoaderCreator,
+} from '@core/dataloader/creators';
+import { ILoader } from '@core/dataloader/loader.interface';
+import { WhiteboardRt } from './whiteboard.rt.entity';
+import { WhiteboardRtService } from './whiteboard.rt.service';
+
+@Resolver(() => IWhiteboardRt)
+export class WhiteboardRtResolverFields {
+  constructor(
+    private whiteboardRtService: WhiteboardRtService,
+    @Inject(WINSTON_MODULE_NEST_PROVIDER)
+    private readonly logger: LoggerService
+  ) {}
+
+  @ResolveField('createdBy', () => IUser, {
+    nullable: true,
+    description: 'The user that created this WhiteboardRt',
+  })
+  async createdBy(
+    @Parent() whiteboardRt: IWhiteboardRt,
+    @Loader(UserLoaderCreator, { resolveToNull: true }) loader: ILoader<IUser>
+  ): Promise<IUser | null> {
+    const createdBy = whiteboardRt.createdBy;
+    if (!createdBy) {
+      throw new EntityNotInitializedException(
```

Comment:
This should return null and log a warning, see the same resolver on the whiteboard field resolver